src/js/components/SkipLinks.js

```diff
@@ -36,15 +35,21 @@ var SkipLinks = React.createClass({
 
   _onBlur: function (event) {
     setTimeout(function () {
-      var _document = React.findDOMNode(this).ownerDocument;
       var skipLinksLayer = this.refs.skipLinksLayer.getDOMNode();
-      var activeElement = _document.activeElement;
+      var activeElement = document.activeElement;
       if (!DOM.isDescendant(skipLinksLayer, activeElement)) {
         this.setState({showLayer: false});
       }
     }.bind(this));
   },
 
+  _onClick: function (destId) {
+    return function (event) {
+      var dest = document.querySelector('#' + destId);
```

Comment:
Sorry, I didn't notice this before. I think document.getElementById(destId) is more efficient than querySelectorAll() when you already have an id.